### DISCUSSION TITLE
Issue with writing speeds

### DISCUSSION THREAD

**Discussion by galgool** [Chia-Network/chia-blockchain#508] [Q&A]
As my old and slow latitude took **15 hours** to plot k32 on an external new hard drive, I purchased a new pc with NVMe just for the chia project.

Running it for the first time, it took **6.5 hours** to plot k32 on this super fast ssd.
In the task Manager, chia writing speeds go up to about 200MB/s and fall to zero.

I ran a performance benchmark and everything seems to works fine.
![image](https://user-images.githubusercontent.com/73280093/98082568-a141f180-1e81-11eb-86e2-63451761a79f.png)

I have no idea what’s wrong.
Would appreciate any help

**Comment by hoffmang9** [comment]
6.5 hours is a very good plotting time for k=32.

**Reply by galgool** [reply]
Yes. But why upgrading to a PC with writing speed 17X faster than my old PC, results in plotting time just 2.3X faster?
Maybe there’s a problem with the GUI?

**Reply by dorkmo** [reply]
im not sure, but i think it would be bc plotting is not 100% writing, so the other factors are still holding it back.

**Reply by galgool** [reply]
Moreover, on this new PC, there’s a Samsung M.2 ssd and a Seagate hard drive:
1. Testing their Writing performance resulted very close to manufacture claims (and of course the ssd is about 15X faster than the hard drive)
2. I’ve ran a k27 (just for time comparison) with the GUI. The ssd was just 4% faster than the hard drive

Anyone understand why and how to fix this?

**Reply by hoffmang9** [reply]
You make the assumption that it’s all writing and it’s not. There are significant sorts that require memory and CPU threads. What you will want to optimize is total TB plotted per day and the way to do that is to plot multiple plots at the same time using your SSD as the temporary working space. You can probably create 3 k32s every 7 hours. Maybe more depending on your RAM and CPU.

**Comment by rlallen-nps** [comment]
@galgool Were you ever able to improve on the performance?

**Reply by galgool** [reply]
Maybe just a little
But today I run 3 parallel plotters on this ssd
